lib/DDG/Spice/NewYorkTimes.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::NewYorkTimes;
+# ABSTRACT: Details on New York Times best sellers
+
+use strict;
+use DDG::Spice;
+
+spice to => 'https://api.nytimes.com/svc/books/v3/lists/best-sellers/history.json?api-key={{ENV{DDG_SPICE_NewYorkTimes_APIKEY}}}&title=$1';
```

Comment:
I registered an API specifically for books, so please rename this to `DDG_SPICE_NYTIMES_BOOKS_APIKEY`